backend/pkg/transport/notifications.go

```diff
@@ -1,15 +1,26 @@
 package transport
 
-import "github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+import (
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+	"time"
+)
 
-// PacketNofitication notifies of an incoming message
+// PacketNotification notifies of an incoming message
 type PacketNotification struct {
-	abstraction.Packet
+	Packet    abstraction.Packet
+	Raw       []byte
```

Comment:
I've talked with FW and now we wont send the raw bytes to the gui, so we can omit this field